pac4j-core/src/main/java/org/pac4j/core/client/Client.java

```diff
@@ -67,4 +60,14 @@
      * @throws HttpAction whether an additional HTTP action is required
      */
     U getUserProfile(C credentials, WebContext context) throws HttpAction;
+
+    /**
+     * <p>Return the logout action (indirect clients).</p>
+     *
+     * @param context the current web context
+     * @param currentProfile the currentProfile
+     * @param targetUrl the target url after logout
+     * @return the redirection
+     */
+    RedirectAction getLogoutAction(WebContext context, U currentProfile, String targetUrl);
```

Comment:
I finally changed the name and the signature of this method to get a `RedirectAction` and delayed its processing if ever we want to implement custom behaviors (with multiple logouts for example).
We need the current profile as well (for OpenID Connect for example) or the computed redirection URL (useful for CAS).